src/mapper.rs

```diff
@@ -134,13 +134,20 @@ pub struct ProguardMapper<'s> {
 impl<'s> From<&'s str> for ProguardMapper<'s> {
     fn from(s: &'s str) -> Self {
         let mapping = ProguardMapping::new(s.as_ref());
-        Self::new(mapping)
+        Self::new(mapping, false)
+    }
+}
+
+impl<'s> From<(&'s str, bool)> for ProguardMapper<'s> {
+    fn from(t: (&'s str, bool)) -> Self {
+        let mapping = ProguardMapping::new(t.0.as_ref());
+        Self::new(mapping, t.1)
     }
 }
 
 impl<'s> ProguardMapper<'s> {
     /// Create a new ProguardMapper.
-    pub fn new(mapping: ProguardMapping<'s>) -> Self {
+    pub fn new(mapping: ProguardMapping<'s>, initialize_param_mapping: bool) -> Self {
```

Comment:
as suggested in chat, I would settle on a default behavior for `new`, and introduce a second `with_param_mapping` constructor. That way you avoid breaking the API.